### PR TITLE
add : added result page

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
1. Added result page of quiz
2. Added user answers and right answers
image

### CODE REVIEW

File: pages/result-page/result-page.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+.red-bg{
+    background-color:var(--red) ;
+}
+
+.optionss button {
+    font-size: 18px;
+    width: 90%;
+    height: 2rem;
+    border-radius: 5px;
+    margin-bottom: 14px;
```
Comment (palash23jain) [inline pages/result-page/result-page.css]: rem/em unit can be preferred here

**Review by palash23jain — APPROVED** [review]
code looks clean.. good to go

File: pages/result-page/result-page.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+.green-bg{
+  background-color:var(--green) ;
+}
+
+.red-bg{
+    background-color:var(--red) ;
+}
+
+.optionss button {
+    font-size: 18px;
```
Comment (kgajriaAgarwal) [inline pages/result-page/result-page.css]: use rem here

File: pages/result-page/result-page.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+.btn-next .btn-restart{
+    font-weight: 600;
+    color:var(--pink-primary2);
+    background-color: var(--white);
+    border:1px solid var(--pink-primary2);
+}
+
+.btn-next .btn-restart:hover{
+    background-color: var(--pink-light2);
```
Comment (kgajriaAgarwal) [inline pages/result-page/result-page.css:46]: improve the color variable naming conventions

**Review by kgajriaAgarwal — CHANGES_REQUESTED** [review]
Kindly do the suggested changes.Rest everything looks good!

**Review by Muzammil-khan710 — APPROVED** [review]
Good to merge Juvita !

File: pages/result-page/result-page.css (hunk 3 above)
Comment (JuvitaSaini09) [reply pages/result-page/result-page.css:46]: i will take care from next,here i have used in lot of files

**Pull request merged**